aeron-cluster/src/main/java/io/aeron/cluster/ConsensusModuleAgent.java

```diff
@@ -1014,10 +1001,6 @@ void onSnapshotRecordingQuery(final long correlationId, final int requestMemberI
 
     void onSnapshotRecordings(final long correlationId, final SnapshotRecordingsDecoder decoder)
```

Comment:
Can this method be removed?

Reply:
Yes, there is still more to be removed as well.